payments/vipps/redirect.md

```diff
@@ -286,6 +208,209 @@ Once the payment is successfully authorized by the payer, she is returned to
 On that page as well as in the `callbackUrl`, you need to perform an HTTP `GET`
 request towards the `id` of the payment to inspect its status.
 
+## Step 3: Get the transaction state
+
+The `GET`request below will give you the transaction state of the payment.
+The `paymentId` used below was provided in the fist step when creating a
+purchase.
+
+{:.code-header}
+**Request**
+
+```http
+GET /psp/{{ payment_instrument }}/payments/{{ page.payment_id }}/ HTTP/1.1
```

Comment:
I don't think `payment_instrument` will have a value here. It's perhaps better to just hard code to `vipps` since this is not an include?

```suggestion
GET /psp/vipps/payments/{{ page.payment_id }}/ HTTP/1.1
```